README.md

```diff
@@ -321,27 +321,27 @@ customizations for your task.
 TEST_DATA_DIR=$(download_bionemo_data single_cell/testdata-20240506 --source $MY_DATA_SOURCE); \
 bionemo-geneformer-recipe \
     --recipe 10m-pretrain \
-    --dest my_config.json \
+    --dest my_config.yaml \
     --data-path ${TEST_DATA_DIR}/cellxgene_2023-12-15_small/processed_data \
     --result-dir ./results
 ```
-> ⚠️ **IMPORTANT:** Inspect and edit the contents of the outputted my_config.json as you see fit
+> ⚠️ **IMPORTANT:** Inspect and edit the contents of the outputted my_config.yaml as you see fit
 
-> NOTE: To pretrain from an existing checkpoint, simply pass in the path --initial-ckpt-path to the recipe command. This will populate the JSON with the correct field to ensure pretraining is initialized from an existing checkpoint.
+> NOTE: To pretrain from an existing checkpoint, simply pass in the path --initial-ckpt-path to the recipe command. This will populate the YAML with the correct field to ensure pretraining is initialized from an existing checkpoint.
```

Comment:
what does it mean to pretrain from the existing ckpt? Ie to resume pretraining or to pretrain from 0 using the other checkpoint config?

Reply:
Updated the language to make this more clear.